systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -93,7 +142,7 @@ public static void waitForIoTProjectReady(Kubernetes kubernetes, IoTProject proj
             }
         }
         if (!isReady) {
-            String jsonStatus = project != null && project.getStatus() != null ? project.getStatus().toString() : "Project doesn't have status";
+            String jsonStatus = project.getStatus() != null ? project.getStatus().toString() : "Project doesn't have status";
```

Comment:
can you implement the toString method in IoTProjectStatus class, check other api-model classes to be aligned with other toString methods in our code